Error: Cannot find module '@aws-sdk/util-dynamodb'

Looks like v3 of the DynamoDB document client now requires @aws-sdk/util-dynamodb as a peer dependency??? Have been getting this error in all functions.

```
{
    "errorType": "Runtime.ImportModuleError",
    "errorMessage": "Error: Cannot find module '@aws-sdk/util-dynamodb'\nRequire stack:\n- /var/task/node_modules/@aws-sdk/lib-dynamodb/dist/cjs/commands/utils.js\n- /var/task/node_modules/@aws-sdk/lib-dynamodb/dist/cjs/commands/BatchExecuteStatementCommand.js\n- /var/task/node_modules/@aws-sdk/lib-dynamodb/dist/cjs/index.js\n- /var/task/index.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js",
    "stack": [
        "Runtime.ImportModuleError: Error: Cannot find module '@aws-sdk/util-dynamodb'",
        "Require stack:",
        "- /var/task/node_modules/@aws-sdk/lib-dynamodb/dist/cjs/commands/utils.js",
        "- /var/task/node_modules/@aws-sdk/lib-dynamodb/dist/cjs/commands/BatchExecuteStatementCommand.js",
        "- /var/task/node_modules/@aws-sdk/lib-dynamodb/dist/cjs/index.js",
        "- /var/task/index.js",
        "- /var/runtime/UserFunction.js",
        "- /var/runtime/index.js",
        "    at _loadUserApp (/var/runtime/UserFunction.js:100:13)",
        "    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)",
        "    at Object.<anonymous> (/var/runtime/index.js:43:30)",
        "    at Module._compile (internal/modules/cjs/loader.js:1068:30)",
        "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)",
        "    at Module.load (internal/modules/cjs/loader.js:933:32)",
        "    at Function.Module._load (internal/modules/cjs/loader.js:774:14)",
        "    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)",
        "    at internal/main/run_main_module.js:17:47"
    ]
}
```

--- a/src/create/package.json
+++ b/src/create/package.json
@@ -6,8 +6,9 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@aws-sdk/client-dynamodb": "^3.13.1",
-    "@aws-sdk/lib-dynamodb": "^3.13.1",
+    "@aws-sdk/client-dynamodb": "^3.19.0",
+    "@aws-sdk/lib-dynamodb": "^3.19.0",
+    "@aws-sdk/util-dynamodb": "^3.19.0",
     "uuid": "^8.3.2"
   },
   "keywords": [],

--- a/src/get/package.json
+++ b/src/get/package.json
@@ -6,8 +6,9 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@aws-sdk/client-dynamodb": "^3.13.1",
-    "@aws-sdk/lib-dynamodb": "^3.13.1"
+    "@aws-sdk/client-dynamodb": "^3.19.0",
+    "@aws-sdk/lib-dynamodb": "^3.19.0",
+    "@aws-sdk/util-dynamodb": "^3.19.0"
   },
   "keywords": [],
   "author": "",

--- a/src/list/package.json
+++ b/src/list/package.json
@@ -6,8 +6,9 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@aws-sdk/client-dynamodb": "^3.13.1",
-    "@aws-sdk/lib-dynamodb": "^3.13.1"
+    "@aws-sdk/client-dynamodb": "^3.19.0",
+    "@aws-sdk/lib-dynamodb": "^3.19.0",
+    "@aws-sdk/util-dynamodb": "^3.19.0"
   },
   "keywords": [],
   "author": "",

--- a/src/remove/package.json
+++ b/src/remove/package.json
@@ -6,8 +6,9 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@aws-sdk/client-dynamodb": "^3.13.1",
-    "@aws-sdk/lib-dynamodb": "^3.13.1"
+    "@aws-sdk/client-dynamodb": "^3.19.0",
+    "@aws-sdk/lib-dynamodb": "^3.19.0",
+    "@aws-sdk/util-dynamodb": "^3.19.0"
   },
   "keywords": [],
   "author": "",

--- a/src/update/package.json
+++ b/src/update/package.json
@@ -6,8 +6,9 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@aws-sdk/client-dynamodb": "^3.13.1",
-    "@aws-sdk/lib-dynamodb": "^3.13.1"
+    "@aws-sdk/client-dynamodb": "^3.19.0",
+    "@aws-sdk/lib-dynamodb": "^3.19.0",
+    "@aws-sdk/util-dynamodb": "^3.19.0"
   },
   "keywords": [],
   "author": "",

